### PR TITLE
Validate claiming file

### PR DESCRIPTION
**Pull request by gabaldon**
This PR validates the claiming file uploaded.

### CODE REVIEW

File: src/components/claiming/UploadFile.vue [hunk 1]
```diff
@@ -71,7 +72,8 @@ export default {
   },
   created() {
     if (this.claimingFileInfo) {
-      this.file.push({ name: 'claiming-process.json' })
+      this.validatedFile(this.claimingFileInfo.info)
```
Comment (tmpolaczyk) [inline src/components/claiming/UploadFile.vue]: This line doesn't do anything?

File: src/components/claiming/Loading.vue [hunk 2]
```diff
@@ -40,6 +37,7 @@ export default {
   },
   watch: {
     async sessionId(value) {
+      console.log(value)
```
Comment (Tommytrg) [inline src/components/claiming/Loading.vue]: Remove log

Comment (aesedepece) [reply src/components/claiming/Loading.vue]: Would love if we could automate this :sweat_smile:

Comment (Tommytrg) [reply src/components/claiming/Loading.vue]: https://eslint.org/docs/rules/no-console.html

**Pull request merged**